NEOSPlus/Logix/Input Devices/TrackerBatteryBase.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using FrooxEngine;
+using FrooxEngine.LogiX;
+
+namespace NEOSPlus.Logix.Input_Devices
+{
+    // TODO: check in multiplayer to make sure this all syncs correctly.
```

Comment:
Has this been checked?

Reply:
yes this has been ill remove the comment